src/Microsoft.AspNet.Mvc.Core/ActionResults/ObjectResult.cs

```diff
@@ -38,6 +38,18 @@ public ObjectResult(object value)
         /// </summary>
         public int? StatusCode { get; set; }
 
+        /// <summary>
+        /// Gets or sets the flag to exclude formatters that match only on the object type.
+        /// <c>false</c> by default.
+        /// </summary>
+        /// <remarks>
+        /// When content-negotiation fails to find a formatter based on the request's 
```

Comment:
Its simpler than this, true it serves to solve this problem but all it does it, allows matching on type if no formatter is found. Now that match can yield a result or not thats subjective. Abstain from writing specifics in the summary. You can add a comment at a appropriate place in the code though.